### PR TITLE
Migrate StaticAppBar to @material-ui/core, shift to styled components

### LINKED ISSUE #2069: Upgrade material-ui to @material-ui/core
**Actual Behaviour**

The current version of Material UI module being used for this project is now deprecated, and should be changed to @material-ui/core module.

**Expected Behaviour**

Use new module with support

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2069 

Changes: 

1. Migrated StaticAppBar to a newer material UI version
2. Make the drawer work, #2095 
3. Shift to styled components, #1669  🎉 

Demo Link: https://pr-2096-1-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

**Comment by fragm3** [comment]
@akshatnitd @harsh-jindal @fossasia/susi-ai please review

File: src/components/StaticAppBar/StaticAppBar.react.js [hunk 1]
```diff
@@ -92,58 +164,42 @@ class StaticAppBar extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      isPopUpMenuOpen: false,
       showAdmin: false,
```
Comment (harsh-jindal) [inline src/components/StaticAppBar/StaticAppBar.react.js]: Remove this state. It is not being set anywhere. Use isAdmin prop to check whether Admin or not.

File: src/components/StaticAppBar/StaticAppBar.react.js [hunk 2]
```diff
@@ -206,258 +262,188 @@ class StaticAppBar extends Component {
...
+          <MenuItem onClick={this.handleMenuClose}>
+            <ListItemIcon>
+              <Info />
+            </ListItemIcon>
+            <ListItemText>
+              <Translate text="About" />
+            </ListItemText>
+          </MenuItem>
+        </Link>
         {this.state.showAdmin === true ? (
```
Comment (harsh-jindal) [inline src/components/StaticAppBar/StaticAppBar.react.js]: Use the isAdmin prop here to check whether Admin or not.

**Comment by fragm3** [comment]
@harsh-jindal made the changes

**Comment by fragm3** [comment]
@akshatnitd all dialog states problem will be solved in https://github.com/fossasia/chat.susi.ai/issues/1954. Will create a PR soon

**Pull request merged**